File: 실습/5장/10_apply/Main.java
public class Main {
    static class User {
        private String name;
        private int age;

        public void setName(String name) {
            this.name = name;
        }

        public void setAge(int age) {
            this.age = age;
        }

        @Override
        public String toString() {
            return "User(name=" + name + ", age=" + age + ")";
        }
    }

    public static void main(String[] args) {
        User user = new User();

        user.setName("철수");
        user.setAge(20);

        System.out.println(user);
    }
}

File: 실습/5장/10_apply/Main.kt
class User {
    var name: String = ""
    var age: Int = 0

    override fun toString(): String {
        return "User(name=$name, age=$age)"
    }
}

fun main() {
    val user = User().apply {
        name = "철수"
        age = 20
    }

    println(user)
}